cluster-density.go

```diff
@@ -42,6 +42,7 @@ func NewClusterDensity(wh *workloads.WorkloadHelper, variant string) *cobra.Comm
 			wh.Metadata.Benchmark = cmd.Name()
 			os.Setenv("JOB_ITERATIONS", fmt.Sprint(iterations))
 			os.Setenv("CHURN", fmt.Sprint(churn))
+			os.Setenv("CHURN_CYCLES", fmt.Sprintf("%v", churnCycles))
```

Comment:
You need to add the CHURN_CYCLES variable to the config file too

Reply:
Tests will likely fail, as that parameter is not in any kube-burner version yet

Reply:
I intentionally did not add it because it would fail our CI (i.e due to dependency: https://github.com/kube-burner/kube-burner/pull/597). But I have verified it on my OCP cluster by changing a config in my local. Since we have verified by running in local, let me get them added into config files too.